src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -90,6 +90,91 @@ public Utf8JsonReader(in ReadOnlySequence<byte> jsonData, bool isFinalBlock, Jso
             }
         }
 
+        /// <summary>
+        /// Constructs a new <see cref="Utf8JsonReader"/> instance.
+        /// </summary>
+        /// <param name="jsonData">The ReadOnlySequence&lt;byte&gt; containing the UTF-8 encoded JSON text to process.</param>
+        /// <param name="options">Defines the customized behavior of the <see cref="Utf8JsonReader"/>
+        /// that is different from the JSON RFC (for example how to handle comments or maximum depth allowed when reading).
+        /// By default, the <see cref="Utf8JsonReader"/> follows the JSON RFC strictly (i.e. comments within the JSON are invalid) and reads up to a maximum depth of 64.</param>
+        /// <remarks>
+        ///   <para>
+        ///     Since this type is a ref struct, it is a stack-only type and all the limitations of ref structs apply to it.
+        ///   </para>
+        ///   <para>
+        ///     This assumes that the entire JSON payload is passed in directly (equivalent to <see cref="IsFinalBlock"/> = true)
+        ///   </para>
+        /// </remarks>
+        public Utf8JsonReader(in ReadOnlySequence<byte> jsonData, JsonReaderOptions options = default)
```

Comment:
Passing `ReadOnlySequence<byte>` in via `in` is demonstrably better for performance, on both Windows and Linux?

Reply:
When we initially designed/added the `Utf8JsonReader` APIs, adding `in` did show perf improvements on the ctor (~8 months ago). I don't know if it still holds true (in the same vein as SequenceReader, but we removed `in` from that).

Is it considered a breaking change to remove `in`? Would it need to go through API review?

Are you recommending I remove the `in` if we don't see a perf benefit? Do you have context around what's changed around `in` recently?

cc @JeremyKuhne

Reply:
> Is it considered a breaking change to remove in?

Yes. So if we're going to do it, we should do it now.

> Would it need to go through API review?

Email would be fine.

> Are you recommending I remove the in if we don't see a perf benefit?

Yes

Reply:
Addressed in a separate PR: https://github.com/dotnet/corefx/pull/38930

Will fix the new overload ones this PR is merged.